app/middlewares/signInValidations.js

```diff
@@ -0,0 +1,12 @@
+const { validateUser } = require('./validations'),
+  error = require('../errors');
+
+exports.checkValidations = (req, res, next) => {
```

Comment:
Do you need an entire file for this ?, also we talk about a generic middleware for validators (not a function for each validation)